<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>661</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>661</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>661</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>101</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>主题</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="textsettings_lb">
      <property name="text">
       <string>自定义主题</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="size_layout" stretch="1,2,2">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>按钮颜色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonCol_btn">
      <property name="text">
       <string>选择颜色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="font_layout" stretch="1,2,2">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <widget class="QLabel" name="font_label">
      <property name="text">
       <string>字体样式</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fontType_btn">
      <property name="text">
       <string>选择字体</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="empty_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>470</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttons_layout">
    <property name="spacing">
     <number>25</number>
    </property>
    <item>
     <widget class="QPushButton" name="save_btn">
      <property name="text">
       <string>保存设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_btn">
      <property name="text">
       <string>取消设置</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="color_layout" stretch="1,2,2">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <widget class="QLabel" name="color_lb">
      <property name="text">
       <string>字体颜色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="choosecol_btn">
      <property name="text">
       <string>选择颜色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="empty">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,2">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <widget class="QLabel" name="back_lb">
      <property name="text">
       <string>背景颜色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="background_btn">
      <property name="text">
       <string>选择颜色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="empty_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="theme_cb">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>chooseTheme()</slot>
  <slot>chooseFontColor()</slot>
  <slot>toPeosonInfo()</slot>
  <slot>chooseFontSize()</slot>
  <slot>chooseFontType()</slot>
 </slots>
</ui>
